solver/src/solver/http_solver/model.rs

```diff
@@ -57,20 +58,20 @@ pub struct TokenInfoModel {
 #[derive(Debug, Serialize)]
 pub struct CostModel {
     pub amount: u128,
-    pub token: String,
+    pub token: H160,
 }
 
 #[derive(Debug, Serialize)]
 pub struct FeeModel {
     pub amount: u128,
-    pub token: String,
+    pub token: H160,
 }
 
 #[derive(Debug, Deserialize)]
 pub struct SettledBatchAuctionModel {
     pub orders: HashMap<String, ExecutedOrderModel>,
     pub uniswaps: HashMap<String, UpdatedUniswapModel>,
-    pub ref_token: String,
+    pub ref_token: H160,
     pub prices: HashMap<String, Price>,
```

Comment:
Shouldn't this also be:
```suggestion
    pub prices: HashMap<H160, Price>,
```

Reply:
I am not sure what you are looking at, but this is H160 already in my code.

Reply:
Introduced already in 99023b3

Reply:
That does not seem to be the case:
```
$ git show 99023b3:solver/src/solver/http_solver/model.rs | grep 'prices:'
    pub prices: HashMap<String, Price>,
```

Reply:
Well I don't know what to tell you, but you are commenting on a line of code that has already been changed to your suggestion.


<img width="944" alt="Screenshot 2021-06-22 at 11 23 33" src="https://user-images.githubusercontent.com/11778116/122899733-5a3f3880-d34c-11eb-8317-3c139e4a0b33.png">

Reply:
![Screenshot 2021-06-22 at 11 25 08](https://user-images.githubusercontent.com/11778116/122899896-835fc900-d34c-11eb-90cc-cd686fd822c0.png)

Reply:
In your second screenshot, line 75 is:
```
    pub prices: HashMap<String, Price>,
```

and I am suggesting it be changed to
```
    pub prices: HashMap<H160, Price>,
```

Reply:
Or right. silly me. Actually I am not so sure about that. Since this is the `SettledBatchAuctionModel` that is returned from the solver. However, I guess if we can interpret the ref_token as H160, then we should also be able to parse the prices key.

Reply:
It should be fine. The keys should get deserialized from their string values, saving the extra work of manually deserializing.